app/mailers/agency_update_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class AgencyUpdateMailer < ApplicationMailer
+
+  def new_agency_update_email
+    @agency = params[:agency]
+    @aur = params[:agency_update_request]
+
+    mail(from: @agency.email, to: 'admin@saf-unite.org', cc: (Proc.new { User.where(role: 'admin').pluck(:email) }).call, subject: 'You got a new organization update request!')
+  end
+
+  def agency_update_approval_email
+    @aur = params[:agency_update_request]
+    attachments.inline['SAFLogo1.png'] = File.read('/Users/nrai/Documents/conectate-carolina/app/assets/images/SAFLogo1.png')
```

Comment:
It would be better if the image is not attached to the email but the email contains a link to the image.

Reply:
@micronix are you referring to the image on the Conectate site? I did not get the part to which image the link should point to?

Reply:
https://user-images.githubusercontent.com/25111094/137979085-a05f507c-7173-411b-b8a9-8a3cf47a3886.mov